### PR TITLE
Update game

### PR DESCRIPTION
**Pull request by loki135**
xzbjzb x

### CODE REVIEW

**Review by loki135 — COMMENTED** [review]
cvscvdf

**Comment by loki135** [comment]
cc

**Review by UKsoftdoc — COMMENTED** [review]
wfjbw

**Review by UKsoftdoc — COMMENTED** [review]
nb,j

**Review by loki135 — COMMENTED** [review]
hb

**Review by loki135 — COMMENTED** [review]
m

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
```
Comment (loki135) [inline index.html:50]: jhjv

File: index.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
...
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (loki135) [inline index.html:50]: kjgj

File: README.md
Comment (loki135) [inline README.md:1]: yufy

**Pull request merged**